content/applications/finance/fiscal_localizations/colombia.rst

```diff
@@ -173,7 +209,7 @@ Information section`:
 - :guilabel:`Gran Contribuyente`: If the company is *Gran Contribuyente* this option should be
   selected.
 - :guilabel:`Fiscal Regimen`: Select the tribute name for the company (`IVA`, `INC`, `IVA e INC`, or
```

Comment:
@samueljlieber do you have an insight on this? Just looking for the best practice here, since these are options the user will select from

Reply:
Yea I agree with @afma-odoo :) since these values are part of a fixed drop-down menu, they should be considered interactive UI text and therefore need a :guilabel:

<img width="571" alt="Screenshot 2024-11-08 at 3 46 16 PM" src="https://github.com/user-attachments/assets/9f651d82-60e5-4cee-94c2-2c0f04c98afd">

However, conversely I'd argue we can use literal formatting (\` \`) for fields that take user input, such as the **Obligaciones y Responsabilidades** field:

<img width="364" alt="Screenshot 2024-11-08 at 3 50 25 PM" src="https://github.com/user-attachments/assets/d652e5a0-67d4-4264-8e55-6e2efb71611d">

Open to thoughts!

Reply:
@samueljlieber Literal should be used when there's no option to select. In this case, there are options so it should be :guilabel: